src/main/java/raisetech/StudentManagement/Student.java

```diff
@@ -0,0 +1,19 @@
+package raisetech.StudentManagement;
```

Comment:
Javaではパッケージ名は小文字のみを使うのが通例です。
```suggestion
package raisetech.studentmanagement;
```